src/OracleService/OracleService.csproj

```diff
@@ -6,6 +6,7 @@
 
   <ItemGroup>
     <PackageReference Include="Neo.ConsoleService" Version="1.0.0" />
+    <PackageReference Include="NeoFS.API" Version="0.0.22" />
```

Comment:
I think the package should be renamed.

Reply:
Do you have any suggestion?

Reply:
`Neo.FileSystem.API`?

Reply:
Will rename.

Reply:
Done. Namespace renamed. PackageId will rename in the future.